.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+# Related Issue
+- issue goes here
+
+# Proposed Changes
+- change 1
+- change 2
```

Comment:
Add a comment to mention what changes have to be done instead of sample checklist

```suggestion
<!-- Insert details about what this whole PR aims to do -->
```

Reply:
I had committed your suggestion